app/models/post.server.ts

```diff
@@ -0,0 +1,17 @@
+type Post = {
+  slug: string;
+  title: string;
+};
+
+export async function getPosts(): Promise<Array<Post>> {
+  return [
+    {
+      slug: "my-first-post",
+      title: "My First Post",
+    },
+    {
+      slug: "90s-mixtape",
+      title: "A Mixtape I Made Just For You",
+    },
+  ];
+}
```

Comment:
style note: we typically do `Post[]` instead of `Array<Post>` but it's not wrong and others might do it that way